packages/core/src/stylable-processor.ts

```diff
@@ -376,7 +366,10 @@ export class StylableProcessor {
                             };
 
                             this.meta.cssVars[cssVar] = property;
-                            this.meta.mappedSymbols[cssVar] = property;
+                            STSymbol.addSymbol({
```

Comment:
Shouldn't it be API on the meta?

Reply:
`addSymbol` is not public and is only available as an internal feature API, we currently only allow getting info from the meta, no editing API